demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -63,9 +63,10 @@ type FormValues = ProductFormDetailsFragment & {
 
 interface FormProps {
     id?: string;
+    showAvailableSince?: boolean;
```

Comment:
imho I would combine it, it gives a bit more context (add Field to the name)

(yes, you could also add Field to non-combined, but then they get pretty long)

Reply:
What do you think about `hiddenFields: ["availableSince"]`?

Reply:
I think the object-boolean version is better because you'll probably have some function detecting if it should be shown or hidden, the return value can directly be assigned. for the string version it's required to manipulate an array based on the function.

```
<Form hideFields={{
    availableSince: hideAvailableSince(user),
}} />
```

vs this?

```
<Form hideFields={[
    ...(hideAvailableSince(user) ? ["availableSince"] : [])
]} />
```

Reply:
Good point, I like the object-boolean version

> If I would implement a form manually I would probably only combine them if there are more than a few. So maybe more then three...

What do you think about this?

One field: No object, direct prop: 

```tsx
<Form hideAvailableSinceField />
```

Two or more fields: Object prop: 

```tsx
<Form
  hideFields={{
    availableSince: true,
    createdAt: false,
  }} 
/>
```

Reply:
For now I implemented the object-prop version.

I already got an idea how to implement the one vs many version, but this would make the code a little bit more complicate. (adding a helper-function to "query" how often a given config-option is set to a specific value in field-config, switch between direct-prop and object-prop code)